packages/ckeditor5-image/src/image/utils.js

```diff
@@ -86,7 +86,7 @@ export function getSelectedImageWidgetAncestor( selection ) {
  * @returns {module:engine/model/element~Element|null}
  */
 
-export function getSelectedImageElementOrAncestor( selection ) {
+export function getSelectedOrAncestorImageElement( selection ) {
```

Comment:
What about `getClosestSelectedImageElement()`?

Reply:
I'm not sure, the image element returned by this function doesn't have to be formally selected (`<image><caption>[Foo]</caption></image>`), so wouldn't this name be misleading?
However, I don't like the name above either. I was thinking about something like `getCorrelatedImageElement` - a bit enigmatic, but still understandable IMO.

Reply:
Maybe then simply `getClosestImageElement()` (`getClosestImageWidget()`)?

Reply:
I liked the previous version better :) Changed to `getClosestSelectedImageElement`.